src/protobufSnapshotting/protobufSnapshotting.csproj

```diff
@@ -34,8 +34,9 @@
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
     <PackageReference Include="CorshamScience.MessageDispatch.Core" Version="2.0.0" />
-    <PackageReference Include="EventStore.Client" Version="5.0.5" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="EventStore.Client.Grpc.Streams" Version="23.3.8" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="protobuf-net" Version="2.4.9" />
```

Comment:
If we go any higher than this version, we get an `Obsolete` warning on `[ProtoMember(1, DynamicType = true)]` due to support for dynamic types not being implemented in higher versions of the package.